app/scripts/controllers/landingPage.js

```diff
@@ -47,4 +61,31 @@ angular.module('openshiftConsole')
       // add the item to recently-viewed. No-op if the dialog is not open.
       addTemplateToRecentlyViewed();
     });
+
+    function dataLoaded() {
+      if (tourConfig && tourConfig.enabled && tourConfig.steps) {
+        if ($routeParams.startTour) {
+          $timeout(function() {
+            $scope.startGuidedTour(resetRouteParams);
+          }, 500);
+        } else if (_.get(tourConfig, 'auto_launch')) {
+          // Check if this is the first time this user has visited the home page, if so launch the tour
+          var viewedHomePageKey = "openshift/viewedHomePage/" + $rootScope.user.metadata.name;
+          if (localStorage.getItem(viewedHomePageKey) !== 'true') {
+            $timeout(function() {
+              localStorage.setItem(viewedHomePageKey, 'true');
+              $scope.startGuidedTour();
+            }, 500);
+          }
+        }
+      }
+    }
+
+    function resetRouteParams() {
+      // Reset the route params in the next digest cycle
+      $timeout(function() {
+        $location.replace();
+        $location.search('startTour', null);
```

Comment:
@jeff-phillips-18 You should be able to add `reloadOnSearch: false` to the route in app.js